modules/op-tkn-condition-management.adoc

```diff
@@ -5,7 +5,7 @@
 = Condition management commands
```

Comment:
Good job! I have a suggestion about the content type attribute. Please implement the suggestion given.

After L2, leave a line. Before L4, add this attribute:
:_mod-docs-content-type: REFERENCE

For more information, see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#content-type-attributes and https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata.

Thanks, @Maharshi-Project.

Reply:
I have updated the file. Could you please verify it?

Reply:
LGTM